Add support for NeoVim

gorillamoe opened the pull request:
Simply added another check to `!has("ruby")` which reads `!has("nvim")`.
NeoVim comes with ruby support by default, so it should be no problem to assume
that ruby is available.

Commit:
Add support for NeoVim

Simply added another check to `!has("ruby")` which reads `!has("nvim")`.
NeoVim comes with ruby support by default, so it should be no problem to assume
that ruby is available.

lukaszkorecki commented:
Hey @BabbaFresh 
If I'm reading this right - your condition is wrong - you're checking if vim you're using doesn't have ruby support AND is not nvim.

Can you also include nvim support in the readme?

Thanks!

BabbaFresh commented:
I can include nvim support in the readme too :+1: 

My condition is not wrong, is it? I added the check where your plugin bails out ..

I simply check if either has not ruby and has not nvim then bail out and do nothing, so my condition should be okay :smile:

Commit:
Add NeoVim support to README

BabbaFresh commented:
Modified README to reflect NeoVim support.

lukaszkorecki commented on README.md:
```diff
@@ -26,6 +26,10 @@ will generate standard TAGS file which later can be used with Vim (standard `:ta
 
+### NeoVim support
+
+[Available since April, 12th 2016](https://github.com/lukaszkorecki/CoffeeTags/pull/57)
```
Can you just add NeoVim to the list in "Editors Supported" section please?

Commit:
Change NeoVim support in README 

Removed separate section about NeoVim support and instead added it to the supported editors section (as requested by the package maintainer Lukasz).

BabbaFresh commented:
Updated the README again (as requested).

lukaszkorecki commented:
@BabbaFresh Perfect! Thank you!
